clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisterModel/RegistrationCommonFormSections.tsx

```diff
@@ -156,83 +260,81 @@ const RegistrationCommonFormSections: React.FC<RegistrationCommonFormSectionsPro
-              />
+              <div className="form-fieldset-wrapper">
+                {endpointInput}
+                <fieldset aria-hidden="true" className="form-fieldset">
+                  <legend className="form-fieldset-legend">
+                    <span>Endpoint</span>
+                  </legend>
+                </fieldset>
+              </div>
             </FormGroup>
             <FormGroup className={spacing.mlLg} label="Bucket" isRequired fieldId="location-bucket">
-              <TextInput
-                isRequired
-                type="text"
-                id="location-bucket"
-                name="location-bucket"
-                value={modelLocationBucket}
-                onChange={(_e, value) => setData('modelLocationBucket', value)}
-              />
+              <div className="form-fieldset-wrapper">
+                {bucketInput}
+                <fieldset aria-hidden="true" className="form-fieldset">
+                  <legend className="form-fieldset-legend">
+                    <span>Bucket</span>
+                  </legend>
+                </fieldset>
+              </div>
             </FormGroup>
             <FormGroup className={spacing.mlLg} label="Region" fieldId="location-region">
-              <TextInput
-                type="text"
-                id="location-region"
-                name="location-region"
-                value={modelLocationRegion}
-                onChange={(_e, value) => setData('modelLocationRegion', value)}
-              />
+              <div className="form-fieldset-wrapper">
+                {regionInput}
+                <fieldset aria-hidden="true" className="form-fieldset">
+                  <legend className="form-fieldset-legend">
+                    <span>Region</span>
+                  </legend>
+                </fieldset>
+              </div>
             </FormGroup>
             <FormGroup className={spacing.mlLg} label="Path" isRequired fieldId="location-path">
-              <Split hasGutter>
-                <SplitItem>
-                  <InputGroupText isPlain>/</InputGroupText>
-                </SplitItem>
-                <SplitItem isFilled>
-                  <InputGroupItem>
-                    <TextInput
-                      isRequired
-                      type="text"
-                      id="location-path"
-                      name="location-path"
-                      value={modelLocationPath}
-                      onChange={(_e, value) => setData('modelLocationPath', value)}
-                    />
-                  </InputGroupItem>
-                </SplitItem>
-              </Split>
+              <div className="form-fieldset-wrapper">
+                {pathInput}
+                <fieldset aria-hidden="true" className="form-fieldset">
+                  <legend className="form-fieldset-legend">
+                    <span>Path</span>
+                  </legend>
+                </fieldset>
+              </div>
```

Comment:
I think this could be cleaner if we add a simple component for this and reuse it for each FormGroup.

Reply:
Even just a small component function at the top of the page if we only need it for this file. If it can be used in other places then we can make it a common component.

Reply:
Good point, thanks @Griffin-Sullivan! Updated this fieldset wrapper to be a reusable component.